src/05-led-roulette/README.md

```diff
@@ -36,8 +36,8 @@ The `no_main` attribute says that this program won't use the standard `main` int
 tailored for command line applications that receive arguments. Instead of the standard `main` we'll
 use the `entry` attribute from the [`cortex-m-rt`] crate to define a custom entry point. In this
 program we have named the entry point "main", but any other name could have been used. The entry
-point function must have signature `fn() -> !`; this type indicates that the function can't return
--- this means that the program never terminates.
+point function must have the signature `fn() -> !`; this type indicates that the function can't
+return - this means that the program never terminates.
```

Comment:
I think the intent with `--` was to create an en-dash, so suggesting that here instead.

```suggestion
return – this means that the program never terminates.
```

Reply:
I see. Changed it.